core/platform/models.py

```diff
@@ -31,17 +31,17 @@
     from mypy_imports import base_models  # pylint: disable=unused-import
 
 # Constant for valid model names.
-NAMES = feconf.VALID_MODEL_NAMES
+NAMES = feconf.ValidModelNames
```

Comment:
Probably this should be Names.

Also is there anything else that uses feconf.ValidModelNames directly? If not then we should probably move the whole eunm here.

Reply:
this enum does get used in wipeout_service.py so I'm unable to move it from feconf.py